tests/unit/helpers/dates/companiDuration.test.js

```diff
@@ -228,7 +228,51 @@ describe('DISPLAY', () => {
     });
   });
 
+  describe('asDays', () => {
+    it('should return duration in days', () => {
+      const companiDuration = CompaniDurationsHelper.CompaniDuration('P3D');
+      const result = companiDuration.asDays();
+
+      expect(result).toBe(3);
+    });
+
+    it('should return duration in days, with hours', () => {
+      const companiDuration = CompaniDurationsHelper.CompaniDuration('P1DT9H');
+      const result = companiDuration.asDays();
+
+      expect(result).toBe(1.375); // 1.375 = 1 + 9 / 24
+    });
+
+    it('should return duration in days, with minutes', () => {
+      const companiDuration = CompaniDurationsHelper.CompaniDuration('P1DT9M');
+      const result = companiDuration.asDays();
+
+      expect(result).toBe(1.00625); // 1.00625 = 1 + 9 / 1440
+    });
+
+    it('should return duration in days, with seconds', () => {
+      const companiDuration = CompaniDurationsHelper.CompaniDuration('P1DT27S');
+      const result = companiDuration.asDays();
+
+      expect(result).toBe(1.0003125); // 1.0003125 = 1 + 27 / 86400
+    });
+  });
```

Comment:
Que penses-tu de tester la duree `P1WT` ? Ajd je ne crois pas que l'on utilise cette duree mais on peut la passer dans le parseur

Reply:
je suis pas pour, si on commence à utiliser les semaines ici, il faudra le faire partout, et je ne vois pas trop de plus-value à ce test, sachant qu'on n'utilise pas ce format de notre côté

Reply:
Dans ce cas se pose la question de definir `this.weeks` dans `DURATION_UNITS` non ?

Reply:
ça va partir cette partie à terme quand on n'autorisera plus que les string ISO